docs/architecture.md

```
@@ -123,10 +125,21 @@ options:
   * Root directory to scan
   * Defaults to the current working directory
 
+* **`-v / --version`**
+
+  * Prints the installed version in the format:
+
+    ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)